android/engine/src/main/java/org/smartregister/fhircore/engine/ui/pin/PinViewModel.kt

```diff
@@ -156,6 +158,8 @@ constructor(
     // _launchDialPad.value = "tel:####"
   }
 
+  // Todo: discuss with ben, whether we need user to redirect to settings or not,
```

Comment:
@dubdabasoduba is there more to discuss here?

Reply:
@pld not sure, @AbdulWahabMemon do you mind elaborating more on this comment. If it's not necessary please delete it. So is line `162` if all concerns have been considered.

Reply:
that comment I added on when Maimoona raised this concern, 
https://onaio.slack.com/archives/C02CAU7V57H/p1644845594155169
as she mentioned that questionnaire and other resources which are get sync with first app initialize can result/cause data issues.
This is the one of the point you mentioned in our last discussion around this ticket, we will decide later whether user can change application when trying relogin ( i.e : variant anc -> ecbis ) 
I am still not sure about this, but option is there too as per Pin Login tickets description, and if user is not checking rememberMe, user will be having that option when relaunch the application